pbm/oplog/oplog.go

```diff
@@ -0,0 +1,70 @@
+package oplog
+
+import (
+	"context"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readconcern"
+
+	"github.com/percona/percona-backup-mongodb/pbm/errors"
+)
+
+var errNoTransaction = errors.New("no transaction found")
+
+func GetOplogStartTime(ctx context.Context, m *mongo.Client) (primitive.Timestamp, error) {
+	ts, err := findTransactionStartTime(ctx, m)
+	if errors.Is(err, errNoTransaction) {
+		ts, err = findLastOplogTS(ctx, m)
```

Comment:
Maybe it should be better to fall back to old criteria if we cannot find such trx:
```
topo.OpTimeFromNodeInfo(inf, true)
```